src/wallet/test/rpc_wallet_tests.cpp

```diff
@@ -1129,15 +1129,15 @@ BOOST_AUTO_TEST_CASE(rpc_z_sendmany_parameters)
     }
 
     try {
-        std::vector<SendManyRecipient> recipients = { SendManyRecipient("dummy",1.0, "") };
+        std::vector<SendManyRecipient> recipients = { SendManyRecipient("dummy", 1*COIN, "") };
```

Comment:
Why are these changes included in this PR?

Reply:
@daira pushed a commit to fix warnings emitted by Clang early on when they first became apparent. It's required because we compile with `-Werror` on Linux.